src/olympia/discovery/tests/test_admin.py

```diff
@@ -1,17 +1,23 @@
 # -*- coding: utf-8 -*-
 from pyquery import PyQuery as pq
+from unittest import mock
 
 import os
+import responses
+
+from rest_framework.reverse import reverse as drf_reverse
```

Comment:
you can use `reverse_ns` in tests (and trim the `v4:` from the start of the view names)